gradle/dependency-locks/apiDependenciesMetadata.lockfile

```diff
@@ -14,13 +14,13 @@ com.amazonaws:aws-java-sdk-support:1.11.424
 com.amazonaws:jmespath-java:1.11.424
 com.atlassian.performance.tools:aws-resources:1.4.0
 com.atlassian.performance.tools:concurrency:1.0.0
-com.atlassian.performance.tools:infrastructure:4.10.0
+com.atlassian.performance.tools:infrastructure:4.12.1
```

Comment:
Didn't help :(

Reply:
Due to https://ecosystem.atlassian.net/browse/JPERF-461 we cannot see from CI, was it a timeout or not?